HTML index with header

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Hello Microverse</title>
+  </head>
```

Comment:
- [ ] In `head`,Let's add link to css file. For ex:- `<link rel="stylesheet" href="index.css">`